### PR TITLE
Vale and markdownlint changes for docs/quickstart.

### PR DESCRIPTION
**Pull request by cgcardona**

### CODE REVIEW

File: .github/styles/Vocab/Products/accept.txt [hunk 1]
```diff
@@ -117,6 +117,7 @@ Grafana
 gwei
 Gwei
 gzip
+hardcoded
```
Comment (dasconnor) [inline .github/styles/Vocab/Products/accept.txt:120]: hard coded

File: .github/styles/Vocab/Products/accept.txt [hunk 2]
```diff
@@ -148,6 +149,8 @@ KYCed
 LevelDB
 liveness
 lol
+locktime
```
Comment (dasconnor) [inline .github/styles/Vocab/Products/accept.txt:152]: lock time

File: .github/styles/Vocab/Products/accept.txt (hunk 1 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

File: .github/styles/Vocab/Products/accept.txt (hunk 2 above)
Comment (cgcardona) [reply .github/styles/Vocab/Products/accept.txt]: Resolved

**Review by dasconnor — APPROVED** [review]

**Review by yulin-dong — APPROVED** [review]

**Pull request merged**